arches/app/src/arches/declarations.d.ts

```diff
@@ -4,6 +4,7 @@ declare module "arches";
 // declare untyped modules that have been added to your project in `package.json`
 declare module "@babel/runtime";
 declare module "@mapbox/geojsonhint";
+declare module "@mapbox/mapbox-gl-draw";
```

Comment:
There are definitions for this on definitely typed, so let's install them as a dev dependency instead of type-ignoring the module.